app/src/pages/SingleContent.vue

```diff
@@ -37,32 +38,38 @@ const isExpiredOrScheduled = computed(() => {
     );
 });
 
-watch(
-    content,
-    async () => {
-        if (!content.value) return;
+watch(content, async () => {
+    if (!content.value) return;
 
-        document.title = isExpiredOrScheduled.value
-            ? `Page not found - ${appName}`
-            : `${content.value.title} - ${appName}`;
+    document.title = isExpiredOrScheduled.value
+        ? `Page not found - ${appName}`
+        : `${content.value.title} - ${appName}`;
 
-        if (isExpiredOrScheduled.value) return;
+    if (isExpiredOrScheduled.value) return;
 
-        const tagIds = content.value.parentTags.concat([content.value.parentId]); // Include this content's parent ID to include content tagged with the parent (if the parent is a tag document).
+    useHead({
+        title: content.value.seoTitle || content.value.title,
+        meta: [
+            {
+                name: "summary",
```

Comment:
```suggestion
                name: "description",
```
the "summary" metadata tag is not supported by Google's indexer